6.recursiveSum.js

```diff
@@ -9,8 +9,29 @@
 
 
 const recursiveSum = (arr) => {
+    // if my arr has no length, return 0
+    if(i >= arr.length) return 0;
+        // add the arr[0] to arr[1] to arr[2] to arr[3] 
+        // to get from arr[0] to arr[1].. ++ 
+        return recursiveSum(arr, i + 1) + arr[i];
 
+    
+    
```

Comment:
Your function would need to take i as an argument and default it to 0. Otherwise i will always be undefined.